suites/reef/rgw/tier-2_ssl_rgw_ecpool_test.yaml

```diff
@@ -166,6 +168,18 @@ tests:
       name: execute Java s3tests
       polarion-id: CEPH-83586289
 
+  - test:
+      abort-on-fail: false
+      config:
+        suite:
+          - maven
+        ssl: true
```

Comment:
@TejasC88 , can you include this test in non-ssl suite as well for better coverage

Reply:
@mkasturi18 added to the regression extended suite as well 
pass log : magna002.ceph.redhat.com:/ceph/ceph-qe-logs/tejas/cephci-run-SUS31D